phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/EnginesFilter.java

```diff
@@ -161,9 +161,11 @@ private boolean isInstalled(EngineCategoryDTO engineCategory, EngineSubCategoryD
      * @return True if the given engine category fulfills the filter, false otherwise
      */
     public boolean filter(EngineCategoryDTO engineCategory) {
-        return searchTerm.map(
-                searchTerm -> engineCategory.getSubCategories().stream().anyMatch(engineSubCategory -> engineSubCategory
-                        .getPackages().stream().anyMatch(version -> version.getVersion().contains(searchTerm))))
+        return searchTerm
+                .map(searchTerm -> engineCategory.getSubCategories().stream()
+                        .anyMatch(engineSubCategory -> engineSubCategory.getPackages().stream()
+                                .anyMatch(version -> version.getVersion().toLowerCase()
+                                        .contains(searchTerm.toLowerCase()))))
```

Comment:
Not really related to this issue but should we use fuzzy search here as well (like for apps)?

Reply:
True